### ISSUE TITLE
Lock window / dim items to sell on AH

### ISSUE THREAD

**Issue by Milky063** [Cidan/BetterBags#158]
### Describe the feature

Can we please add a Lock anchor to the window. I keep clicking on the window and move it by accident. Also when you open the AH, is there a way to dim the items that we can't sell. Allows us to determine/see which items can be sold on the AH.  

### Describe the implementation

^

**Comment by Cidan** [comment]
Yes, a lock anchor mode will make a return in the future. Can you make a different issue for the AH one please? It helps me track each individual work item.

Thanks :)
